store/api/store.api

```diff
@@ -1,11 +1,3 @@
-public abstract interface class com/dropbox/android/external/store4/DiskRead {
-	public abstract fun read (Ljava/lang/Object;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
-}
-
-public abstract interface class com/dropbox/android/external/store4/DiskWrite {
-	public abstract fun write (Ljava/lang/Object;Ljava/lang/Object;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
-}
-
```

Comment:
Thank you for this!

Reply:
No problem, glad to help. 

If there are other issues blocking this going to beta, let us know please. I would like to introduce this to our work project, but they are not very fond of alpha versions.